input/yxgxk.yml

```diff
@@ -0,0 +1,234 @@
+tracers:
```

Comment:
Isn't there a more obscure file name? What `yxgxk` stands for? `ymilca` cross `g`? cross `kappa`?

Reply:
Yes. It's the name Nick gave to his project https://github.com/nikfilippas/yxgxk. It'll be easier to identify that way, I think.